### ISSUE TITLE
Statistics percentage of correctly answered cards

### ISSUE THREAD

**Issue by GraefinZahl** [ankidroid/Anki-Android#4326]
###### Research

_Enter an [ x ] character to confirm the points below:_

[ x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[ x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[ x ] I have searched for similar existing issues here and on the user forum
###### Reproduction Steps
1. Go to statistics and open “ANSWER BUTTONS“ tab.
2. 
3. 
###### Expected Result

Before the update one could see the percentage of correctly answered cards per card type (new, young, mature).
###### Actual Result

Now the info is gone.
I enabled advanced statistics in the menu and went through the others tab, but couldn't find it anymore. Was it really removed or am I just blind? 
I really appreciated this piece of statistics though.
###### Debug info

Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

AnkiDroid Version = 2.6

Android Version = 4.4.2

ACRA UUID = 78b748fb-1f57-4659-b910-a8d233661742

Sorry for any mistakes, English is not my native language.
Thanks in advance and many greetings,
Mia

**Comment by eginhard** [comment]
What do you actually see when selecting the "Answer buttons" tab? Could you post a screenshot? I'm not sure right now if anything specific was changed there from the last version, but that information should still be shown there.

**Comment by hssm** [comment]
It's the result of [this](https://github.com/ankidroid/Anki-Android/issues/3506). They were removed because the desktop client doesn't show them inside the graph so this makes us more consistent. 

In the future we might make this information visible in the overview tab.

**Comment by stooone** [comment]
+1 for this. I really miss this info.

**Comment by studentjunk** [comment]
I agree. having that information is very helpful. if this is not desired on the graph, the text should be made available to be more consistent with the desktop version.

**Comment by gajewsk2** [comment]
This was the most useful information for me in the graphs section. Can we please bring g them back somehow. As an exclusive ankidroid user, you can't effectively adjust your deck's interval modifiers without this information.

**Comment by kanjieater** [comment]
Can we reopen the issue? This was the most useful information for me in the graphs section. As an exclusive ankidroid user, you can't effectively adjust your deck's interval modifiers without this information.

**Comment by mikehardy** [comment]
Well, we are always "listening" even though we don't have a lot of developer time. I'll reopen this but since I'm not personally familiar I'm going to mark it as "needs triage" to see what it would mean to bring the text in to the overview tab.

**Comment by GraefinZahl** [comment]
By the way @mikehardy 
I didn't mean to show the information in the overview tab.
In the PC version the percentage of correctly answered card can be seen below the graph, which is shown under the "answer buttons" tab.
Originally, that was also the case for the App version, but it was removed somehow..

**Comment by david-allison-1** [comment]
I'd like to take this on for AnkiDroid 2.12

**Comment by david-allison-1** [comment]
Relevant Anki Code: https://github.com/ankitects/anki/blob/816cec4a58d8d9c845eb94fe626ff229189c0ec4/pylib/anki/stats.py#L723-L748
